pkg/filesystem/path/resolve.go

```diff
@@ -38,6 +39,12 @@ func (rs *resolverState) push(scopeWalker ScopeWalker, path string) error {
 		path = stripOneOrMoreSlashes(path)
 		absolute = true
 	}
+	if runtime.GOOS == "windows" {
```

Comment:
Please don't make changes along these lines.

Keep in mind that `pkg/filesystem/path` is supposed to be a platform agnostic path resolution library. Even though nobody should do anything as crazy as this, in theory it should be supported to run bb_worker on a Linux box, expose its build directory over a Samba server to a Windows box, and have bb_runner running there. Or the opposite.

`path.Resolve()` should just implement an abstract path resolver that does the same thing on every platform. If this is insufficient, then we should have discussions how to fix that up. Maybe we should rename it to `path.ResolveUNIX()`, and have a separate `path.ResolveWindows()`?

Taking a step back: are there actually any places where we call `path.Resolve()` against a Windows style path that contains drive letters? As far as I know, the only field in all of REv2 that contains a Windows style path that we need to process is `argv[0]`.

Reply:
Handled much better in other commits and PRs. It was never meant for this PR.